ludwig/api.py

```diff
@@ -308,8 +308,8 @@ def close(self):
         """Closes an open LudwigModel (closing the session running it).
         It should be called once done with the model to release resources.
         """
-        if self.model is not None:
-            self.model.close_session()
+        # TODO(travis): determine whether this method is still needed in TF2
```

Comment:
Yes all the API still needs to be adapted to TF2, close on and train online and other things may not be needed or may work differently

Reply:
I suppose what's not clear to me is: since there's no equivalent of a session in TensorFlow 2, is there still some kind of global state manager that needs to be manually closed, or can we just re-init TensorFlow through `tf.config`.  I think this will require some investigation to determine.

Reply:
My best guess: if a model is loaded for prediction purposes, there should be no problem of global state, while for training there's the optimizer state etc. So maybe we could consider having two methods, one that's actually `load` and one that is `resume_training` which takes care of recreating the global state.